test/dotnet-migrate.Tests/GivenThatIWantToMigrateTestApps.cs

```diff
@@ -26,6 +26,7 @@ public class GivenThatIWantToMigrateTestApps : TestBase
         // https://github.com/dotnet/sdk/issues/73 [InlineData("TestAppWithLibrary/TestApp", false)]
         [InlineData("TestAppWithRuntimeOptions")]
         [InlineData("TestAppWithContents")]
+        [InlineData("TestAppWithSigning")]
```

Comment:
We do a test that the properties that get passed csc for signing are populated. 

How do we test that the assembly is signed?